src/video/movies.cpp

```diff
@@ -289,8 +420,12 @@ void ffmpeg_stop_movie()
 void upload_yuv_texture(uint8_t **planes, int *strides, uint32_t num, uint32_t buffer_index)
 {
 	uint32_t upload_width = strides[num];
-	uint32_t tex_width = num == 0 ? movie_width : movie_width / 2;;
-	uint32_t tex_height = num == 0 ? movie_height : movie_height / 2;
+    // Use these for yuv444:
+    uint32_t tex_width = movie_width;
+	uint32_t tex_height = movie_height;
+    // Use these for yuv420:
+	//uint32_t tex_width = num == 0 ? movie_width : movie_width / 2;;
+	//uint32_t tex_height = num == 0 ? movie_height : movie_height / 2;
```

Comment:
Please remove the dead code if not meant to be used.

Reply:
Done.